components/Checkbox.js

```diff
@@ -41,6 +41,7 @@ const Checkbox = ({ className, id, labelText, onChange, ...other }) => {
           <Icon
             className="bx--checkbox-checkmark"
             description="checkmark"
+            alt="checkmark"
```

Comment:
Have created a prop "iconDescription" to replace the hard coded value "checkmark"